app/src/main/java/io/apicurio/registry/storage/impl/sql/CommonSqlStatements.java

```diff
@@ -105,15 +105,15 @@ public String selectGlobalRuleByType() {
      */
     @Override
     public String deleteGlobalRule() {
-        return "DELETE FROM globalrules r WHERE r.tenantId = ? AND r.type = ?";
+        return "DELETE FROM globalrules WHERE tenantId = ? AND type = ?";
```

Comment:
Does mssql not support this syntax or is this personal preference?

Reply:
Looking at the rest of the PR, this seems like a change that's not needed.  If that's the case, could we revert it?

Reply:
The problem is that this DELETE statement gives an error with mssql, as described here: https://stackoverflow.com/questions/11005209/why-cant-i-use-an-alias-in-a-delete-statement

I thought since removing the alias doesn't break or complexes for pgsql it could be done like that.

Reply:
I see, thank you.  Is the problem only for DELETE statements?

Reply:
Yes.